### DISCUSSION TITLE
Required Kicad version

### DISCUSSION THREAD

**Discussion by GioCC** [yaqwsx/Pinion#2] [Q&A]
What is the minimum / maximum KiCad version required?
I use the nightly development version and it does not seem to work (for one, it does not correctly find the pcbnew Python library, which is in a different location with regard to KiCad 5.1.10).
NB A similar indication should also be specified for pcbdraw.

**Comment by yaqwsx** [comment]
Currently, only 5.1 is supported but support for nightly is on the roadmap (for both tools).

**Reply by GioCC** [reply]
That's really great, thanks!

**Reply by yaqwsx** [reply]
Pinion already works with nightly. It also works with nightly on Windows - the installation is just a little bit clumsy (I will make instructions once I figure out how to make it simple enough for the users).
